src/encoded/tests/test_schemas.py

```diff
@@ -15,8 +15,8 @@
 ]
 
 
```

Comment:
You can't call a fixture, but `master_mixins` was being called, so I've split it out into a callable function and a fixture.